tests/test_templates.py

```diff
@@ -30,78 +31,81 @@ def find_envs():
 @pytest.mark.parametrize("env", find_envs())
 def test_env(env, monkeypatch):
     monkeypatch.setattr(flow.FlowProject, "_store_bundled", gen._store_bundled)
+    monkeypatch.setattr(env, "scheduler_type", FakeScheduler)
```

Comment:
Copy whatever comment you write in `test_project.py` here, it's useful for future devs to know why the `FakeScheduler` is important (as opposed to the `MockScheduler`).